env.groovy

```diff
@@ -0,0 +1,5 @@
+env.DB_URL="Pman Status, Pfioh Pull"
+env.TO="parsingh@redhat.com"
```

Comment:
Ideally you wouldn't commit this but rather add it as it's deployed with a configmap

Reply:
Moved it to secrets